Frameworks/include/Starboard.h

```diff
@@ -205,4 +205,19 @@ inline CAPoint3D operator*(float f, const CAPoint3D& v) {
     return CAPoint3D(v.x * f, v.y * f, v.z * f);
 }
 
+// Wrapper for C++/WinRT types when you need a trivial default constructor
+template <typename T>
+class TrivialDefaultConstructor : public T {
+public:
+    TrivialDefaultConstructor() : T(nullptr) { }
+
+    auto operator=(const T& other) {
```

Comment:
sorry, I misread this as `operator==`. Ignore.

---
In reply to: [109243318](https://github.com/Microsoft/WinObjC/pull/2382#discussion_r109243318) [](ancestors = 109243318)